R/supervised-model-development.R

```diff
@@ -26,6 +26,7 @@ SupervisedModelDevelopment <- R6Class("SupervisedModelDevelopment",
     grainTest = NA,
     prevalence = NA,
     factorLevels = NA,
+    imputeVals = NA,
```

Comment:
Do we want to add an imputeVals parameter to SupervisedModelDevelopmentParams so that the user can actually choose which values to use for imputation? Or do you think it's enough to provide them with the imputeDF function which they can use on their own if  they want to use custom values?